build.sbt

```diff
@@ -19,13 +19,13 @@ resolvers in ThisBuild += "Funky-Repo" at "http://lynbrookrobotics.com/repo"
 
 lazy val sharedDependencies = if (System.getenv("NATIVE_TARGET") == "ARM32") { 
   Def.setting(Seq(
-    "org.typelevel"  %%% "squants"  % "1.3.0"
+    "org.typelevel"  %% "squants"  % "1.3.0"
```

Comment:
@AnkitLakkapragada these should not have been changed; they break our ability to compile to JS and Native.